src/main/java/uk/gov/hmcts/juror/api/moj/xerox/letters/ConfirmLetter.java

```diff
@@ -36,14 +36,13 @@ protected void setupWelsh() {
         addData(LetterDataType.DATE_OF_LETTER, 18);
         addData(LetterDataType.COURT_LOCATION_CODE, 3);
         addData(LetterDataType.WELSH_COURT_NAME, 40);
-        addData(LetterDataType.COURT_NAME, 40);
-        addData(LetterDataType.BUREAU_NAME, 35);
-        sharedBureauSetup();
-        addData(LetterDataType.BUREAU_POSTCODE, 10);
+        addData(LetterDataType.BUREAU_NAME, 40);
+        addBureauAddress();
         addData(LetterDataType.BUREAU_PHONE, 12);
         addData(LetterDataType.BUREAU_FAX, 12);
         addData(LetterDataType.WELSH_DATE_OF_ATTENDANCE, 32);
         sharedSetup();
+        addData(LetterDataType.BUREAU_SIGNATORY, 30);
```

Comment:
Assume this is intended as not in the spec?

Reply:
This aligns to the heritage application we have a task post go live to review the spec misalignments